RELEASES.md

```diff
@@ -1,3 +1,353 @@
+Version 1.7.0 (2016-03-03)
+==========================
+
+Language
+--------
+
+* Soundness fixes to the interactions between associated types and
+  lifetimes, specified in [RFC 1214], [now generate errors][1.7sf] for
+  code that violates the new rules. This is a significant change that
+  is known to break existing code, so it has emitted warnings for the
+  new error cases since 1.4 to give crate authors time to adapt. The
+  details of what is changing are subtle; read the RFC for more.
+
+Libraries
+---------
+
+* Stabilized APIs:
+  [`Path::strip_prefix`][] (renamed from relative_from),
+  [`path::StripPrefixError`][] (new error type returned from strip_prefix),
+  [`Ipv4Addr::is_loopback`],
+  [`Ipv4Addr::is_private`],
+  [`Ipv4Addr::is_link_local`],
+  [`Ipv4Addr::is_multicast`],
+  [`Ipv4Addr::is_broadcast`],
+  [`Ipv4Addr::is_documentation`],
+  [`Ipv6Addr::is_unspecified`],
+  [`Ipv6Addr::is_loopback`],
+  [`Ipv6Addr::is_multicast`],
+  [`Vec::as_slice`],
+  [`Vec::as_mut_slice`],
+  [`String::as_str`],
+  [`String::as_mut_str`],
+  `<[T]>::`[`clone_from_slice`],
```

Comment:
Is this a good place for a note about new behavior in `clone_from_slice`? It should say that `clone_from_slice` now requires the two slices to be the same length (and panics otherwise).